apps/root/src/pages/earn/components/strategies-table/index.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Strategy } from 'common-types';
+import { EarnPosition, Strategy } from 'common-types';
```

Comment:
I like all the generalization here, tho I'm a bit afraid it's gonna bite us in the ass later when we want to extend it a bit more.

For now lets keep is as you made it